resources/views/control-panel/user-management-list.blade.php

```diff
@@ -2,7 +2,7 @@
     <x-jet-section-title>
         <x-slot name="title">{{ __('Manage Users') }}</x-slot>
         <x-slot name="description">
-            {{ __('Manage user details, create new users, impersonate users, or delete users.') }}
+            {{ __('Manage user details, create new users, view server as specific user, or delete users.') }}
```

Comment:
Doesn't really seem to fit since you're not viewing a server, the actual logic is impersonating as a specific user within the website/app, a server is what the website is hosted on, not the actual software.

The code that handles the logic is also defined using the word impersonating so it makes sense to keep everything consistent so people know what they need to look for if they wanna find the files and edit them.

The code in question: https://github.com/Senither/ShareX-Media-Server/blob/master/app/Http/Middleware/Impostering.php

Reply:
Reverted it, I'm new to pull requests, so is this fine?